metadata-ingestion/source_overview.md

```diff
@@ -0,0 +1,27 @@
+# Sources
+
+:::tip Find Integration Source
+Please see our **[Integrations page](https://datahubproject.io/integrations)** to browse our ingestion sources and filter on their features.
+:::
+
+Sources refers to the data systems that we are extracting metadata from.
+The `Sources` tab on the left in the sidebar shows you all the sources that are available for you to ingest metadata from. For example, we have sources for [BigQuery](https://datahubproject.io/docs/generated/ingestion/sources/bigquery), [Looker](https://datahubproject.io/docs/generated/ingestion/sources/looker), [Tableau](https://datahubproject.io/docs/generated/ingestion/sources/tableau) and many others.
+
+In general, the source will be defined at the top of the recipe like below.
+
+```yaml
+source: <source_name>
+config: ...
```

Comment:
this doesn't look right - should be this

also docusaurus lets you set "filenames" on code blocks - I think putting a filename of "my_recipe.yml" will help make this more clear

```
# my_recipe.yml
source:
  type: <source name>
  config:
    option_1: value
    ...
```